examples/vision/xray_classification_with_tpus.py

```diff
@@ -153,12 +142,16 @@ def process_path(file_path):
 Load and format the test data as well.
 """
 
-test_list_ds = tf.data.Dataset.list_files(str(GCS_PATH + "/chest_xray/test/*/*"))
-TEST_IMG_COUNT = test_list_ds.cardinality().numpy()
-test_ds = test_list_ds.map(process_path, num_parallel_calls=AUTOTUNE)
-test_ds = test_ds.batch(BATCH_SIZE)
+test_images = tf.data.TFRecordDataset(
+    "gs://download.tensorflow.org/data/ChestXRay2017/test/images.tfrec"
+)
+test_paths = tf.data.TFRecordDataset(
+    "gs://download.tensorflow.org/data/ChestXRay2017/test/paths.tfrec"
+)
+test_ds = tf.data.Dataset.zip((test_images, test_paths))
 
-print("Testing images count: " + str(TEST_IMG_COUNT))
+test_ds = test_ds.map(process_path, num_parallel_calls=AUTOTUNE)
+test_ds = test_ds.batch(1)
```

Comment:
Why 1?

Reply:
I chose 1 so that `model.predict()` would only run on a single image for the run inference section at the very bottom.

Reply:
We first compute evaluation metrics on the test data via `model.evaluate(test_ds)` -- so we should use a large batch size.

Then you can draw one batch and call predict on it, and visualize results for the first sample